tox.ini

```diff
@@ -37,7 +37,7 @@ deps =
 commands =
     # install detection requirements later as mmdet requires torch installed
     # Torch dependency is added to force torch version to remain the same
-    pip install -e .[detection]
+    pip install -e .[training]
```

Comment:
I wonder what `training` means here?

Reply:
It installs both classification and detection. Please refer to setup.py.

Reply:
Thanks for the reply. Is there any specific use-case, where classification and detections are specifically needed together?

Reply:
No, but segmentation task installation will be added to [training] soon. If using [full] which anomaly installation is included, dependency re-check takes very long time, so separated mpa training tasks.

Reply:
I see, would it be an idea to name it `mpa` maybe?

Reply:
Yes, it might be better. I revised.